### PR TITLE
Update dataset conversion script for active learning

### PR DESCRIPTION
**Pull request by naga-karthik**
This PR updates the dataset conversion script by adding an option to include axial zurich images to the training set. 

More importantly, the script checks for the intersection between the training set subjects and the corrected axial subjects. Only the subjects common to the traning are used to enable proper comparison with the models trained _without_ active learning.

### CODE REVIEW

File: dataset-conversion/convert_bids_to_nnUNetv2_all_sci_data.py [hunk 1]
```diff
@@ -64,40 +65,32 @@ def get_parser():
     parser.add_argument('--split', nargs='+', required=True, type=float, default=[0.8, 0.2],
                         help='Ratios of training (includes validation) and test splits lying between 0-1. Example: '
                              '--split 0.8 0.2')
+    # input yaml file containing list of axial subjects to include for active learning
+    parser.add_argument('--include-axial', type=str, default=None,
+                        help='Path to yaml file containing list of axial subjects to include for active learning.')
```
Comment (valosekj) [inline dataset-conversion/convert_bids_to_nnUNetv2_all_sci_data.py:70]: How is this YAML file obtained? Downloaded from HTML QC or created manually?

**Review by valosekj — APPROVED** [review]
LGTM! Thank you!

File: dataset-conversion/convert_bids_to_nnUNetv2_all_sci_data.py (hunk 1 above)
Comment (naga-karthik) [reply dataset-conversion/convert_bids_to_nnUNetv2_all_sci_data.py:70]: I would say it is created semi-manually. The HTML QC also contains info such as ses-xx, contrast, etc. But this yaml file simply contains the subject names to be included. Example for a few subjects:

```
CORR_LESION:
- sub-zh01
- sub-zh03
- sub-zh05
- sub-zh10
- sub-zh14
- sub-zh25
- sub-zh30
- sub-zh32
- sub-zh33
- sub-zh34
- sub-zh38
- sub-zh41
- sub-zh42
```

Comment (valosekj) [reply dataset-conversion/convert_bids_to_nnUNetv2_all_sci_data.py:70]: Okay!

**Comment by naga-karthik** [comment]
Great, thanks for the quick review! Merging

**Pull request merged**